guides/working-with-starcounter/unload-reload/README.md

```diff
@@ -0,0 +1,140 @@
+# Using Unload/Reload to Modify Database Schema
+
+When making changes to the database schema of an application, the change has to be made in the database and in the application. This is accomplished by following these steps:
+
+1. Unload the database

```

Comment:
Please write more specific, like "Unload the database to an SQLite dump" and "Modify the schema in the dump".